wordpress-mobile/WordPress-Android
# Reference the fixed self-hosted page parent FluxC build

Fixes #8428.

#### To test

1. Add a self-hosted site without Jetpack to the app.
2. Open Site Pages and, if needed, publish multiple pages.
3. Select the ellipsis menu > Set Parent for one of the pages.
4. Select a parent page.
5. Wait for the action to complete
6. Verify the page parent has changed accordingly
7. Pull to refresh
8. Verify the page parent is still the same
